lib/teyu/perfect_initializer.rb

```diff
@@ -0,0 +1,20 @@
+require "teyu/fast_initializer"
+require "teyu/generic_initializer"
+
+module Teyu
+  # This initializer tries the fast initializer, and then fallback to generic initializer if failed.
+  class PerfectInitializer
```

Comment:
@sonots 
There is nothing perfect.
How about `Teyu::Initializer` ?

Reply:
@takanamito lol

Reply:
renamed